src/NzbDrone.Core/MediaCover/MediaCoverResizer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ImageResizer;
+using NzbDrone.Common.Disk;
+
+namespace NzbDrone.Core.MediaCover
+{
+    public interface IMediaCoverResizer
+    {
+        void ResizeMediaCover(string sourceFileName, string resizedFileName, int height);
```

Comment:
lets make the names more generic since it doesn't do anything specific to our domains, so 

`MediaCoverResizer` -> `ImageResizer `

`ResizeMediaCover` -> `Resize`